rosette/lib/util/ws-server.rkt

```diff
@@ -17,34 +17,31 @@
 ;; modified software. See http://www.gnu.org/licenses/lgpl-3.0.txt for
 ;; more information.
 
-(require racket/match)
```

Comment:
I’d prefer we didn’t make changes to this file, because it’s copy-pasted from the rfc6455 package, and keeping it as-is makes the difference obvious.

Reply:
Done.